docs/specs/tripgo.openapi.yaml

```diff
@@ -947,6 +947,16 @@ paths:
         schema:
           type: integer
           default: 8
+      - name: strictDrive
```

Comment:
Rather than adding a new parameter and option would be to re-use the `neverAllowModes` parameter and also make it work for `modes=me_car&neverAllowModes=wa_wal`. It's semantically equivalent and might be simpler to understand for API users why that wouldn't work but `modes=me_car&strictDrive=true` would work.

If we do it that way, we only need to adjust the documentation for `neverAllowModes`.